apps/etterlatte-behandling/src/main/kotlin/behandling/sjekkliste/Sjekkliste.kt

```diff
@@ -6,7 +6,7 @@ data class Sjekkliste(
     val id: UUID,
     val kommentar: String? = null,
     val kontonrRegistrert: String? = null,
-    val onsketSkattetrekk: Int? = null,
+    val onsketSkattetrekk: String? = null,
```

Comment:
Er dette for å kunne støtte både prosent og beløp?

Reply:
Aye!